src/components/Tabs/Tabs.module.scss

```diff
@@ -10,9 +10,27 @@ $theme-grey: #d4d4d5;
     cursor: pointer;
 }
 
+.featureFlagTabButton {
+    background: white;
+    border: none;
+    padding: 12px 8px 8px;
+    font-size: 0.8em;
```

Comment:
Could you check whether em suits better or rem here

Reply:
Checked it for all dimensions. There is no visible change for such a short value.